include/sgct/readconfig.h

```diff
@@ -20,6 +20,9 @@ namespace sgct {
 
 [[nodiscard]] std::string serializeConfig(const config::Cluster& cluster);
 
+[[nodiscard]] std::string serializeConfig(const config::Cluster& cluster,
+    const config::GeneratorVersion& genVersion);
```

Comment:
I would say, make it `config::GeneratorVersion genVersion` instead and then make it a `std::optional` with a `std::nullopt` default value.. don't want to force everyone to think about specifying it when they don't want to

Reply:
Yeah, this makes it satisfy the tests that have no generator version as well.